env_configs

```diff
@@ -0,0 +1 @@
+/home/tools/config
```

Comment:
Is this required?

Reply:
no, its an artefact the tg alias creates, i'll remove